### PR TITLE
Update tests to handle RiakTS 1.4 changes.

### PR DESCRIPTION
**Pull request by christophermancini**
Cell descriptions will include more fields than the original 5 going forward, so I updated the assertion to be >= making it backwards compatible. Also changed test timestamp due to key conflict with other tests.

### CODE REVIEW

File: tests/TimeSeriesTrait.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ protected static function tableDefinition($table_name = "")
 
     protected static function populateKey()
     {
-        static::$now = new \DateTime("@1443806900");
+        static::$now = new \DateTime("@1443816900");
```
Comment (lukebakken) [inline tests/TimeSeriesTrait.php]: Why the change? Just curious.

Comment (christophermancini) [inline tests/TimeSeriesTrait.php]: I had a conflict with another client lib which was causing a test to fail.

**Comment by christophermancini** [comment]
@lukebakken my bad, I forgot to update the unit tests the first time. They have been fixed and Travis should be happy soon. :)

**Comment by lukebakken** [comment]
:ship: :it:
